pkg/webkitgtk2

```diff
@@ -0,0 +1,85 @@
+[mirrors]
+https://webkitgtk.org/releases/webkitgtk-2.20.3.tar.xz
+
+[vars]
+filesize=16623456
+sha512=69a7b27e294de6fd43240ce1d314cd1ccdbf29674c97967674459fd02720b31fa53e5412efbd93dc1e805d38e38feb7fe9a12cdd9c472471676b8db3c11734de
+pkgver=1
+
+[deps]
+cairo
+enchant
+gtk+3
+harfbuzz
+libsoup
+libtasn1
+libwebp
+
+[deps.host]
+cmake
+gperf
+perl
+python
+ruby
```

Comment:
lol, maybe we should ask them to add some js scripts too, so we can add node to the mix

Reply:
honestly a little surprising there is no bash dependency lol